src/easier68k/core/util/parsing.py

```diff
@@ -1,5 +1,32 @@
 # Parsing utils
 
+def from_str_util(command: str, parameters: str) -> (str, list):
+    """
+    Util method for from_str
+    Splits the command into both parts, gets the normalized size
+    and splits the parameters
+
+    >>> from_str_util('MOVE.B', '#123, D0')
+    ('B', ['#123', 'D0'], ['MOVE', 'B'])
```

Comment:
Could use some more tests for 1-parameter commands, possibly no-parameter commands (even though we wouldn't need it for that), also for commands both with and without sizes (if necessary)